liqd_product/apps/partners/templates/partner_base.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'base.html' %}
+
+{% load userindicator static i18n %}
+
+{% block after_header %}
+<img src="{{partner.logo}}" alt="Logo" />
+            {{partner.name}}
+<header class="header">
```

Comment:
This header should have an `aria-label` attribute (e.g. "platform header") to explain the difference to the partner header